README.md

```diff
@@ -69,15 +69,25 @@ chmod -R a+x ./scripts
 ./scripts/dev.local install-hooks
 ```
 
-## Run the apA
+## Run the app
 
-To run the app, simply call the startup script!
+First time running, run these four commands:
```

Comment:
We should have this under `##Setup`

How about creating `scripts/setup` that does all of this for you? Including the git hooks thing?